dogs.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="./styles/master.css">
+    <title>Here are all of our Dogs!</title>
+  </head>
+
+  <body>
+    <header>
+      <img src="images/adoptadog-logo.svg" width="200" alt="adoptadog logo">
+      <p>Seattle's premiere dog adoption agency</p>
+
+      <img src="images/cart-icon.svg" width="24" alt="shopping cart icon">
+      <p>$1234.56</p>
+      <button type="button">Checkout</button>
+
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="dogs.html">Dogs</a></li>
+          <li><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+    </header>
+
+
+    <main>
+      <h2>Here are all of our Dogs!</h2>
+        <section>
```

Comment:
You did a great job structuring this page. You'll want to create a class, maybe called dog-cards and use it to create your grid. You can use flexbox to create the grid, or css grid. Checkout this reference on using flexbox: https://css-tricks.com/dont-overthink-flexbox-grids/

This will allow you to create the grid of tiles seen in the PDF example we provide in your project older